src/main.rs

```diff
@@ -12,10 +14,20 @@ struct CTRConfig {
     author: String,
     description: String,
     icon: String,
-    target_path: String,
+    target_path: PathBuf,
     cargo_manifest_path: PathBuf,
 }
 
+impl CTRConfig {
+    fn path_3dsx(&self) -> OsString {
+        self.target_path.with_extension("3dsx").into()
```

Comment:
One downside of this, specifically for unit tests, is that the filename contains a hash, so we'll end up with multiple copies on the device as dependencies change. Maybe it's worth special-casing those to name it `${lib}-${test}.3dsx` or something like that?